core-bundle/tests/Functional/SimpleTokenExtensionTest.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Tests\Functional;
+
+use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;
+
+class SimpleTokenExtensionTest extends KernelTestCase
```

Comment:
This does not seem like a functional test that requires a running system. A regular unit test would suffice IMHO.

Reply:
How would this work? 

see https://github.com/contao/contao/blob/46f5475e4d4707631c1f173c7429bc3dd1820f39/core-bundle/tests/Functional/app/config/services.yml

Reply:
I like the additional functional test :)

Reply:
We do not have to test whether the Symfony container registers the tagged providers IMHO. Testing the DI container is Symfony's responsibility and not ours.

Reply:
Imo the test documents nicely how this feature can/should be used (and ensures it's intended usage works). I like this in general (if reading the tests explains the code) but I can drop it as well if you prefer.

Reply:
@contao/developers WDYT?

Reply:
As discussed in Mumble on July 2nd, this should be a unit test. 🙈

Reply:
Done